### ISSUE TITLE
[MultiColumnComboBox] column borders are misaligned when virtualization is enabled

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#3189]
**Describe the bug**
Enabling virtualization, misalign the column borders, and render the last column data with 3 dots, even if there is enough space for the content:

![26 01 2021_16 20 16_REC](https://user-images.githubusercontent.com/33104481/105857001-83080f80-5ff2-11eb-9852-4a6b53d302fb.png)

**To Reproduce**
https://stackblitz.com/edit/angular-n7xpkc

Reported in private support thread: **1503791**, **1562301**, **1608661**, **1619944**, **1662216**

**Comment by yanmariomenev** [comment]
In addition to the issue if _media_ is added to the column, the alignment between the header and column is larger:
```
<kendo-combobox-column
            media="(min-width: 300px)"
                field="title"
                title="Job Title"
                [width]="200"
            >
</kendo-combobox-column>
```
![chrome_EmvOocDtqv](https://user-images.githubusercontent.com/49076727/151159037-6620a07b-4190-4b25-93d8-02e77e1db003.png)

**Comment by stnikolova** [comment]
The reported issue that 3 dots are rendered in the last column is not reproducible in the provided StackBlitz example. Please specify if any special environment requirements are present for the issue to be present.

**Comment by erothfield** [comment]
If you change one or more of the last name values in the sample to be longer than will fit, you can see there is an issue with misalignment of the columns and header.  The column overflow handling does not seem to work with virtualization.

![image](https://user-images.githubusercontent.com/11988359/167239996-b855de5d-9696-4bc7-84f1-61abd011188a.png)

**Comment by gklevi** [comment]
We face the same issue.  
Any info about the version this issue will be fixed?  
Currently, we are forced to use a drop down list in our project but would prefer to use the multi column combo box as we need more data on selection of an item.

**Comment by thomasponzo-dev** [comment]
I also have the same problem

**Comment by Raisolution** [comment]
Fix available in `v17.1.1`
